packages/jira-adapter/src/product_settings/data_center/data_center.ts

```diff
@@ -250,6 +250,14 @@ const PLUGIN_URL_PATTERNS: UrlPattern[] = [
     httpMethods: ['get'],
     url: '/rest/api/3/issuetypescheme/project.+',
   },
+  {
+    httpMethods: ['post'],
+    url: '/rest/api/3/notificationscheme',
+  },
+  {
+    httpMethods: ['put', 'delete'],
+    url: '/rest/api/3/notificationscheme/\\d+',
```

Comment:
```suggestion
    url: '/rest/api/3/notificationscheme/.+',
```
We want the request to get to the plugin even when the id is invalid